src/state/SceneFooterState.tsx

```diff
@@ -161,7 +163,7 @@ export const FooterAccordionEventService = () => {
 
   useSharedEvent(scrollEndEvent, () => {
     'worklet'
-    if (Platform.OS === 'android') {
+    if (isAndroid) {
```

Comment:
Not related to this review, but it is a bit suspect that you have to do a `runOnJs` here for Android. This implies you'll always cross-the-bridge every time the user's finger lifts off the screen on scroll.

Reply:
The runOnJS is because there is no `setTimeout` on the UI worklet thread.

Reply:
I may need a recap on the advantage of the setTimeout. What happens if we remove it? Can you try and see if perf improves